src/widgets/ActivityDetails/model/hooks/useEntityComplete.ts

```diff
@@ -19,21 +19,35 @@ export const useEntityComplete = (props: Props) => {
 
   const { getGroupInProgressByIds } = activityModel.hooks.useActivityGroupsInProgressState()
 
-  const completeActivityFlowAndRedirect = () => {
+  const completeEntityAndRedirect = () => {
     entityCompleted({
       appletId: props.appletDetails.id,
       entityId: props.flowId ? props.flowId : props.activityId,
       eventId: props.eventId,
     })
 
+    const entityCompletedState = {
+      isAnswersSubmitted: true,
+    }
```

Comment:
I see the idea. If it is just used to render a toast then we should thing of a different idea as this one violates SOLID

Reply:
Done